packages/flutter/lib/src/widgets/async.dart

```diff
@@ -274,28 +274,28 @@ typedef Widget AsyncWidgetBuilder<T>(BuildContext context, AsyncSnapshot<T> snap
 /// of the following snapshots that includes the last one (the one with
 /// ConnectionState.done):
 ///
-/// * `new AsyncSnapshot<int>(ConnectionState.waiting, null, null)`
-/// * `new AsyncSnapshot<int>(ConnectionState.active, 0, null)`
-/// * `new AsyncSnapshot<int>(ConnectionState.active, 1, null)`
+/// * `AsyncSnapshot<int>.withData(ConnectionState.waiting, null)`
```

Comment:
Because if they were code, they should really be `const`, but can't now due to an analyzer bug. And they are not so much code as an indication of values.